ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/TransactionMetaData.kt

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+
+@DoNotImplement
+@CordaSerializable
+interface TransactionMetaData{
```

Comment:
I think the meta will be an internal concern at least for now, so probably should start out in corda-runtime-os. The same is probably true for PrivacySalt, but that is tied into the legacy stuff so we should leave here

Reply:
Thanks. I have moved TransactionMetadata into runtime-os.